web/forms.py

```
@@ -1,6 +1,8 @@
 import re
+from typing import ClassVar, Dict, List
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)